website/pages/api-docs/system/internal-counters.mdx

```diff
@@ -130,3 +130,166 @@ $ curl \
   "auth": null
 }
 ```
+
+## Client Count
+
+This endpoint returns the number of clients per namespace, as the sum of active entities and non-entity tokens.
+An "active entity" is a distinct entity that has created one or more tokens in the given time period.
+A "non-entity token" is a token with no attached entity ID.
+
+A time period may be specified; otherwise it reports on a default reporting period, such as the
+previous twelve calendar months. Reports are only available with month granularity, after each month
+has completed. The response included the actual time period covered, if a query covers more time, or
+less time, than requested.
+
+All namespaces will be included, that are children of the namespace where the request was made.
```

Comment:
I think this could be a bit more clear too. "All namespaces will be included, such as" or something like that?

Reply:
I attempted to rewrite it.